roles/deploy_user/tasks/main.yml

```diff
@@ -73,12 +73,18 @@
   when: running_on_server
 
 - name: deploy_user | install deploy github ssh config
-  ansible.builtin.copy:
-    src: files/ssh_config
-    dest: "/home/{{ deploy_user }}/.ssh/config"
-    owner: "{{ deploy_user }}"
-    group: "{{ deploy_user }}"
+  ansible.builtin.blockinfile:
+    path: "/home/{{ deploy_user }}/.ssh/config"
+    insertbefore: BOF
     mode: 0644
+    create: true
+    marker_begin: BEGIN_GITHUB
+    marker_end: END_GITHUB
+    block: |
+      Host github.com
+        HostName github.com
```

Comment:
The PR title says GitLab. This says GitHub. I'm confused.

Reply:
This is the GitHub block, the one in the lib_jobs task is for GitLab. 

I had to give the two blocks unique markers so that the `blockinfile` module wouldn't just keep overwriting them back and forth, and would add both of them.

Reply:
@acozine  - added an explanation to the PR description.